packages/main/src/TabContainer.js

```diff
@@ -915,7 +919,11 @@ class TabContainer extends UI5Element {
 		});
 
 		if (!this._getEndOverflow().hasAttribute("hidden")) {
-			focusableTabs.push(this._getEndOverflow().querySelector("[ui5-button]"));
+			if (this.shadowRoot.querySelector("slot[name=overflowButton]")) {
+				focusableTabs.push(this.shadowRoot.querySelector("slot[name=overflowButton]"));
```

Comment:
The change seems correct. Would be a bit nicer if you introduce getters for these, but might be personal preference, so I will let that up to you:

get startExtOVerflowBtnDOM
-> this.shadowRoot.querySelector("slot[name=startOverflowButton]"

get extOverflowBtnDOM
-> this.shadowRoot.querySelector("slot[name=overflowButton]")
 
 get startOverflowBtnDOM
 -> this._getStartOverflow().querySelector("[ui5-button]")
 
get overflowBtnDOM
-> this._getEndOverflow().querySelector("[ui5-button]")